presto-function-namespace-managers/src/main/java/com/facebook/presto/functionNamespace/mysql/FunctionNamespaceDao.java

```diff
@@ -50,23 +50,23 @@
     void createFunctionNamespacesTableIfNotExists();
 
     @SqlUpdate("CREATE TABLE IF NOT EXISTS <sql_functions_table> (\n" +
-            "  id bigint(20) NOT NULL AUTO_INCREMENT,\n" +
-            "  function_id varchar(700) NOT NULL,\n" +
+            "  function_id_hash varchar(128) NOT NULL,\n" +
+            "  function_id text NOT NULL,\n" +
             "  version bigint(20) unsigned NOT NULL,\n" +
             "  catalog_name varchar(128) NOT NULL,\n" +
             "  schema_name varchar(128) NOT NULL,\n" +
             "  function_name varchar(256) NOT NULL,\n" +
-            "  parameters varchar(40000) NOT NULL,\n" +
-            "  return_type varchar(256) NOT NULL,\n" +
+            "  parameters text NOT NULL,\n" +
+            "  return_type text NOT NULL,\n" +
             "  routine_characteristics text NOT NULL,\n" +
             "  body mediumtext,\n" +
             "  description text,\n" +
             "  deleted boolean NOT NULL DEFAULT false,\n" +
             "  delete_time TIMESTAMP NULL DEFAULT NULL,\n" +
-            "  create_time TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,\n" +
-            "  update_time TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,\n" +
+            "  create_time TIMESTAMP NULL DEFAULT CURRENT_TIMESTAMP,\n" +
+            "  update_time TIMESTAMP NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,\n" +
             "  PRIMARY KEY (id),\n" +
```

Comment:
Do we have test for this? We probably should?

Reply:
Tests were failing.